bpf/cpu/cpu.bpf.c

```diff
@@ -932,7 +987,7 @@ int walk_user_stacktrace_impl(struct bpf_perf_event_data *ctx) {
         return 1;
       }
 
-      LOG("[error] previous_rip should not be zero. This can mean that the read failed, ret=%d while reading @ %llx.", err, previous_rip_addr);
+      LOG("[error] previous_rip should not be zero. This can mean that the read failed, ret=%d while reading previous_rip_addr", err);
```

Comment:
Could we print the address when we have it?

Reply:
Do you mean `previous_rip_addr`? It is always printed slightly earlier in the code- the reason I removed it here is because since we only use it for x86, the if-else condition earlier introduced some scoping issues with the verifier- happy to find ways around it but maybe in a follow up PR?

Reply:
Oh sounds good!